libbpf-tools/softirqs.bpf.c

```diff
@@ -21,8 +22,18 @@ __u64 counts[NR_SOFTIRQS] = {};
 __u64 time[NR_SOFTIRQS] = {};
 struct hist hists[NR_SOFTIRQS] = {};
 
+static bool is_target_cpu() {
+        if (targ_cpu < 0)
+	        return true;
+
+	return bpf_get_smp_processor_id() == (u32)targ_cpu;
```

Comment:
Do we need (u32) here? I guess not. Let us do
  targ_cpu == bpf_get_smp_processor_id()
which is more code friendly.